cli/utilities/packages.py

```diff
@@ -82,7 +84,7 @@ def install(self, pkg, nogpgcheck=False, env_vars={}):
         out = self.execute(sudo=True, long_running=True, cmd=cmd)
         if isinstance(out, dict):
             if not verify_execution_status(out, pkg):
-                raise PackageError("Failed to install package '{pkg}'")
+                raise PackageError(f"Failed to install package '{pkg}'")
```

Comment:
This change should not be part of this PR

Reply:
Understood, I’ll remove this change from the PR.